mne/viz/evoked.py

```diff
@@ -94,29 +95,34 @@ def _butterfly_onselect(xmin, xmax, ch_types, evoked, text=None):
         evoked_fig = plt.gcf()
         evoked_fig.canvas.draw()
         evoked_fig.canvas.flush_events()
-    times = evoked.times
-    xmin *= 0.001
+
     minidx = np.abs(times - xmin).argmin()
-    xmax *= 0.001
     maxidx = np.abs(times - xmax).argmin()
     fig, axarr = plt.subplots(1, len(ch_types), squeeze=False,
                               figsize=(3 * len(ch_types), 3))
+
     for idx, ch_type in enumerate(ch_types):
+        if ch_type not in ('eeg', 'grad', 'mag'):
+            continue
         picks, pos, merge_grads, _, ch_type = _prepare_topo_plot(
-            evoked, ch_type, layout=None)
-        data = evoked.data[picks, minidx:maxidx]
+            info, ch_type, layout=None)
+        this_data = data[picks, minidx:maxidx]
         if merge_grads:
             from ..channels.layout import _merge_grad_data
-            data = _merge_grad_data(data)
-            title = '%s RMS' % ch_type
+            this_data = _merge_grad_data(this_data, psd=psd)
+            title = '%s mean' % ch_type if psd else '%s RMS' % ch_type
         else:
             title = ch_type
-        data = np.average(data, axis=1)
+        this_data = np.average(this_data, axis=1)
         axarr[0][idx].set_title(title)
-        plot_topomap(data, pos, axes=axarr[0][idx], show=False)
-
-    fig.suptitle('Average over %.2fs - %.2fs' % (xmin, xmax), fontsize=15,
-                 y=0.1)
+        vmin = min(this_data) if psd else None
+        vmax = max(this_data) if psd else None  # All negative for dB psd.
```

Comment:
You can just use `viridis` or some other sequential (non-diverging) colormap. In the case that the data is not diverging we should probably set vmax/vmin as min/max of data still